dev/dll/XamlControlsResources.cpp

```diff
@@ -74,6 +78,13 @@ void XamlControlsResources::UpdateSource()
 
             hstring releasePrefix;
 
+#ifdef USE_INTERNAL_SDK
+            if (is21H1OrHigher && useNewVisual)
```

Comment:
>&& useNewVisual [](start = 31, length = 15)

remove this. If compiled with new SDK, we have 21h1_themesoureces anyway

Reply:
I wasn't using 21h1_themesoureces_2dot5.xaml and was using the identical 19h1_themesoureces_2dot5.xaml. I changed that and 21h1_themesoureces_2dot5.xaml is now used.